### ISSUE TITLE
Unable to add show due to error with theTVDB

### ISSUE THREAD

**Issue by cbinder4** [SickChill/SickChill#5867]
AA ValueError: invalid literal for int() with base 10: ''
AA self.startyear = int(myShow.firstAired.split('-')[0])
AA File "C:\SickRage\SickRage\sickbeard\tv.py", line 821, in loadFromIndexer
AA newShow.loadFromIndexer()
AA File "C:\SickRage\SickRage\sickbeard\show_queue.py", line 401, in run
AA ValueError: invalid literal for int() with base 10: ''
AA self.startyear = int(myShow.firstAired.split('-')[0])
AA File "C:\SickRage\SickRage\sickbeard\tv.py", line 821, in loadFromIndexer
AA newShow.loadFromIndexer()
AA File "C:\SickRage\SickRage\sickbeard\show_queue.py", line 401, in run
AA Traceback (most recent call last):
2020-01-14 09:49:04 ERROR SHOWQUEUE-ADD :: [105c4c8] Error trying to add show: invalid literal for int() with base 10: ''
AA ValueError: invalid literal for int() with base 10: ''
AA self.startyear = int(myShow.firstAired.split('-')[0])
AA File "C:\SickRage\SickRage\sickbeard\tv.py", line 821, in loadFromIndexer
AA newShow.loadFromIndexer()
AA File "C:\SickRage\SickRage\sickbeard\show_queue.py", line 401, in run
AA Traceback (most recent call last):
2020-01-14 09:49:04 ERROR SHOWQUEUE-ADD :: [105c4c8] Unable to add show due to an error with theTVDB: invalid literal for int() with base 10: ''
2020-01-14 09:49:04 INFO SHOWQUEUE-ADD :: 359606: Unable to find the show in the database
2020-01-14 09:49:04 INFO SHOWQUEUE-ADD :: Starting to add show by ShowDir: E:\Brave New World

I was able to add shows immediately after updating to v2020.01.11-1 but have my first update after that seemed to cause me to begin having this issue.  I've just updated again to v2020.01.13-3 but still have the issue.  I'm currently unable to add any shows.

**Comment by cbinder4** [comment]
SickChill Info:
Branch: master
Commit: 105c4c8fdca6060bc33b5837aec1cc81d7f0d79a
Version: v2020.01.13-3
Database Version: 44.1
  Python Version:2.7.10 (default, May 23 2015, 09:44:00) [MSC v.1500 64 bit (AMD64)]
  SSL Version:OpenSSL 1.0.2a 19 Mar 2015
  OS:Windows-8-6.2.9200
  Locale:en_US.cp1252

**Comment by miigotu** [comment]
@cbinder4 because Brave New World (2019) has no premiere/start date on tvdb. I will make a fix so it doesn't error

**Comment by cbinder4** [comment]
Thanks... yes, I used to get a notification that it could not add for this reason.  The whole change to adding Existing Shows is something I am not a fan of so far.  I normally create directories ahead of time for shows that have been announced and periodically add them this way.  Before, my list of shows populated quickly and I went through them one by one selecting the proper shows. Now, it takes a long time to populate because it seems to be searching for them all first.  That would be OK and a nice add'l feature, except there is no option to change which show comes up if it chooses the wrong one by default.  Is it possible to add an option to select a different show or a toggle in the settings to allow shows to be searched one by one like it used to be?
